Add to author response content more sample data.

gnott opened the pull request:
Regarding issue https://github.com/elifesciences/XML-mapping/issues/88, I wanted to add some more types of content that may appear in a decision letter or author response.

I was happy to find many of the content types were already represented in the kitchen sink XML.

Here I'm suggesting to add some additional content to the author response that I didn't see was represented in any `<sub-article>`:

- `<disp-formula>`
- unicode character: ϕ
- `<list>` with a nested `<list>`, with `@list-type` of `order` and `roman-lower` (ones I've observed in samples so far)
- apostrophe character '
- `<sub>`
- `<sup>`
- escaped ampersand `&amp;`
- unmatched angle bracket `&lt;`
- `<italic>`

How these are represented is up to revising and suggestions. This may inform how a decision letter parser library XML output sample may look, which may be loosely based on this kitchen sink XML.

gnott committed:
Add to author response content more sample data.

Melissa37 commented:
@FAtherden-eLife can you review and merge this?

Thanks!

fred-atherden commented:
Changes made in this look fine. Holding merging pending discussion in https://github.com/elifesciences/XML-mapping/issues/89, as it _might_ be worth adding the `@id`s before merging.

Melissa37 commented:
Agree - suggest adding IDs to equations too then merge

gnott committed:
Add id to the disp-formula in the author response.

gnott committed:
Add id to mml:math tags inside inline-formula.

gnott commented:
Finished a couple more commits. The first, where I neglected to add `@id` to the `<disp-formula>` I added to the author response, the values are now present. The second, I added an `@id` to the `<mml:math>` tags that are inside `<inline-formula>` tags.

I can leave this PR open for another check before merging, @Melissa37 or @FAtherden-eLife if you please - thank-you!

fred-atherden commented:
Looks good - thanks @gnott.
